### PR TITLE
Hotfix org

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: packages/publisher-dapp/src/Pages/OrganizationSetup/validationConstraints.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { validBlockChainAdressEquality } from "../../Utils/validation";
 export const submitOrganizationCostraints = {
   id: { presence: { allowEmpty: false } },
   name: { presence: { allowEmpty: false } },
-  website: { url: true },
```
Comment (raamb) [inline packages/publisher-dapp/src/Pages/OrganizationSetup/validationConstraints.js:6]: Why this change?

File: packages/publisher-dapp/src/Services/Redux/actionCreators/organizationActions.js [hunk 2]
```diff
@@ -406,7 +407,6 @@ export const publishToIPFS = uuid => async dispatch => {
       dispatch(loaderActions.stopAppLoader());
       throw new APIError(error.message);
     }
-    dispatch(loaderActions.stopAppLoader());
```
Comment (raamb) [inline packages/publisher-dapp/src/Services/Redux/actionCreators/organizationActions.js:409]: We don't need this anymore?

File: packages/publisher-dapp/src/Pages/OrganizationSetup/validationConstraints.js (hunk 1 above)
Comment (Vivek205) [reply packages/publisher-dapp/src/Pages/OrganizationSetup/validationConstraints.js:6]: the `url` was basic URL validation in the validtor.js. It is too  primitive , it is throwing error even if the URL contains a uppercase in it. This causes trouble while publishing the org.

the `validURL` is the custom function added to validtor and it tests the URL against a regex pattern. It is more forgiving.

File: packages/publisher-dapp/src/Services/Redux/actionCreators/organizationActions.js (hunk 2 above)
Comment (Vivek205) [reply packages/publisher-dapp/src/Services/Redux/actionCreators/organizationActions.js:409]: Stopping loader here causing a gap between two loaders in the screen, which will allow the user to click on publish again even if there is an ongoing publish process.

Nevertheless, the loader is stopped if there is any error.
